api/policies.go

```diff
@@ -0,0 +1,78 @@
+package api
+
+import (
+	"context"
+	"github.com/gofrs/uuid"
+
+	"encoding/json"
+	"github.com/ONSdigital/dp-permissions-api/models"
+	"github.com/ONSdigital/log.go/v2/log"
+	"net/http"
+)
+
+func (api *API) PostPolicesHandler(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Consider making this name singular instead of plural. Having the plural name implies that multiple polices can be created at once. (or is that the intention?)
```suggestion
func (api *API) PostPolicyHandler(writer http.ResponseWriter, request *http.Request) {
```